staging/src/kubevirt.io/api/testdata/README.md

```diff
@@ -0,0 +1,155 @@
+                              "tscFrequency": -12
+                            },
+                -           "virtualMachineRevisionName": "virtualMachineRevisionNameValue"
+                +           "virtualMachineRevisionName": "virtualMachineRevisionNameValue",
+                +           "runtimeUser": 0
+                          }
+                        }
+                        """
+                  )
+                
+            compatibility.go:422: yaml differs
+            compatibility.go:423:   (
+                        """
+                        ... // 237 identical lines
+                                  pciAddress: pciAddressValue
+                                  readonly: true
+                -               floppy:
+                -                 readonly: true
+                -                 tray: trayValue
+                                io: ioValue
+                                lun:
+                        ... // 341 identical lines
+                          qosClass: qosClassValue
+                          reason: reasonValue
+                +         runtimeUser: 0
+                          topologyHints:
+                            tscFrequency: -12
+                        ... // 22 identical lines
+                        """
+                  )
+                
+```
+
+The above output shows that for VirtualMachineInstance:
+1. api-field: `spec.domain.devices.disks.floppy` was dropped. [ref-1](https://github.com/kubevirt/kubevirt/issues/2016)[ref-2](https://github.com/kubevirt/kubevirt/pull/2164)
+2. api-field: `status.runtimeUser` field was added[ref-3](https://github.com/kubevirt/kubevirt/pull/6709)
+
+
+## REVIEWERS GUIDE
+
+With any change in go structs of the APIs, the corresponding JSON and YAML files will be updated.
+
+When decoding, round-tripping identical bytes, or decoding identical objects from JSON/YAML, failures trigger detailed error outputs.
+These errors specify the differences found during the comparison.
+
+Some cases involve adding a new non-pointer fields to existing API types. These fields serialize zero values, which may lead to additional fields being output during the round-tripping process with data from previous releases.
+
+This discrepancy causes round-trip tests to fail, as the output includes unexpected fields.
+
+To address this, an expected data file (<group>.<version>.<kind>_after_roundtrip.[json|yaml]) is placed next to the serialized data file from a previous release.
+
+This file contains the expected data after the round-tripping process, accounting for any new fields or changes.
+
+These after_roundtrip files can be generated by running the failing tests with the environment variable UPDATE_COMPATIBILITY_FIXTURE_DATA=true. This updates the compatibility fixture data to reflect the current expected output.
+
+Using this API reviewers can say if changes in current version will break older clients upon upgrade.
+
+
+### //TO-DO
```

Comment:
I suggest you rename it to "Open Issues" or something similar. In the current form, it looks incomplete.